### PR TITLE
Support React 0.14

### PR DESCRIPTION
**Pull request by marcelpanse**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,12 +47,16 @@
...
     "webpack": "^1.10.1",
     "webpack-dev-server": "^1.6.6"
   },
+  "dependencies": {
+    "highlight.js": "^8.4.x"
+  },
   "peerDependencies": {
-    "highlight.js": "^8.4.x",
-    "react": "*"
+    "react": "^0.14.0-rc1",
```
Comment (tarmolov) [inline package.json]: `0.14.2` is out. Let's don't use `rc` in deps.

**Comment by tarmolov** [comment]
@akiran could you take a look to a pr? This error is very annoying.

File: package.json (hunk 1 above)
Comment (marcelpanse) [inline package.json]: Done

Comment (akiran) [inline package.json]: Yes, I will review and merge this PR

**Comment by korczis** [comment]
Any updates on this? The warning is really annoying.

**Comment by marcelpanse** [comment]
He still didn't merge it. 

In the meanwhile you can use my repository like this in your package.json:
`"react-highlight": "git+https://github.com/marcelpanse/react-highlight.git"`

**Comment by aldendaniels** [comment]
@Dean177 - To my knowledge, peer dependencies are being updated, not deprecated.

See [this NPM blog post](http://blog.npmjs.org/post/110924823920/npm-weekly-5):

> We will also be changing the behavior of peerDependencies in npm@3. We won’t be automatically downloading the peer dependency anymore. Instead, we’ll warn you if the peer dependency isn’t already installed. This requires you to resolve peerDependency conflicts yourself, manually, but in the long run this should make it less likely that you’ll end up in a tricky spot with your packages’ dependencies

File: index.js [hunk 2]
```diff
@@ -1 +1 @@
-module.exports = require('./lib');
\ No newline at end of file
+module.exports = require('./src/index.jsx');
```
Comment (akiran) [inline index.js]: index.js should not import ./src/index.jsx.

npm package will have jsx to js transformed files in lib folder. which should be imported

File: package.json [hunk 3]
```diff
@@ -47,12 +47,16 @@
...
+    "react-dom": "^0.14.0-rc1"
```
Comment (akiran) [inline package.json]: still using rc1.
Please change it to latest version

**Comment by akiran** [comment]
Merged PR #11  which also added support for react 0.14

**Pull request closed without merging**